### ISSUE TITLE
Prisma deploy Error: "No 'Authorization' header provided."

### ISSUE THREAD

**Issue by pedrojardim** [prisma/prisma#4703]
I have a Prisma server and a Postgres DB, all of them on Azure. So, I have deployed a prisma service to this Prisma server. He create everything all right, all tables. Then I import my data, and everything works fine. But, when I try to make a new Deploy, I get an error.
On Prisma Server the error is: 

{"key":"error/handled","requestId":"local:cjxdmeh1e012x07493kvdqlgs","payload":{"exception":"com.prisma.deploy.schema.AuthFailure: No 'Authorization' header provided.","query":"{\n        listProjects {\n          name\n        }\n      }","variables":"{}","code":"3015"
....
"message":"No 'Authorization' header provided."}}

And on my prisma CLI it shows this error: 
```
(Code: 504): {"response":{"error":"<html>\r\n    <head>\r\n        <title> Server Error </title>\r\n    </head>\r\n    <body>\r\n        <font color =\\\"#aa0000\\\">\r\n            <h2>Server Error.</h2>\r\n        </font>\r\n\r\n        There was an unexpected error in the request processing.\r\n    </body>\r\n</html>","status":504}
```
and

Error: spawn ENAMETOOLONG

I can make the prisma reset and prisma delete commands, but the prisma deploy throws the error.

We also try to make a local deploy, having a local doker with prisma server and local postgres. This environment had no problem, the deploy works every time. 
But the problem reapers when we set the local prisma server to point to our postgres on azure. 

About my PRISMA_MANAGEMENT_API_SECRET, it is set on my docker-compose.yml.
I also set PRISMA_MANAGEMENT_API_SECRET on my CLI. This is so, that we can execute listProjects query at /management, using a token generated by cluster-token with prisma CLI.
So, we think the PRISMA_MANAGEMENT_API_SECRET is set, because cluster-token is working.

any ideas about what could be wrong?

 - Connector: Postgres
 - Prisma Server: 1.34
 - `prisma` CLI: 1.34
 - OS: Linux

**Comment by pantharshit00** [comment]
Can you please try this command again with `export DEBUG=*` and see whether Prisma is including a jwt token in the request.

Also, can you please give me an estimated size of you datamodel. If possible share the datamodel with me.

**Comment by pedrojardim** [comment]
Sorry for that too long to respond.

I just find out that the log about the "'Authorization' header provided." happens all the times that I made a deploy, and it appears to be a miss mach error. Because the procedure works, even when this log of error appearing. 

And I found what my problem was.

I had a Prisma Server on a Docker compose,  and a PostresSql DB. Both on azure cloud. And with this architecture the latency between the prisma server and the Postgres was too high. So the second deploy, the one that supposes to just update the model, was getting a time out, and the server dont respond with a good error log, the CLI show this generic server error, and the DB change its not made. BUT, some times the DB model update happens. What we fund weird. 

So, to test this hypothesis we include the Postgres inside our Docker compose. And just like that everything work just fine, and very, very fast. 

So our problem is when the postgres is not inside our docker.
